packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -10,7 +10,7 @@ To learn more about DevTools, check out the
 
 ## General updates
 
-TODO: Remove this section if there are not any general updates.
+* Added a new feature for deep link validation, supporting deep link web checks on Android. - [#6935](https://github.com/flutter/devtools/pull/6935)
```

Comment:
Can we add a screenshot of this tool here?

Reply:
Discussed offline with Chun Heng. Will leave up to you whether you want to soft launch this or announce more loudly. These notes are displayed in DevTools when a user opens a new Devtools version for the first time.

Reply:
I think this is fine to be a simple one line description just for the record. We can prepare more graphics and description once we are ready to do a public announcement.